packages/graphql/src/api-v6/queryIRFactory/resolve-tree-parser/graphql-tree/where.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Args for `where` in nested connections (with edge -> node) */
+export type GraphQLWhereArgs = LogicalOperation<{
```

Comment:
Shall we remove the `args` suffix? I don't think it adds any value
```suggestion
export type GraphQLWhere= LogicalOperation<{
```